### PR TITLE
Filtered CLI, Installer and Longrun tests for upgrade

### PR DESCRIPTION
**Pull request by jyejare**
Test Files updated with upgrade decorator in :
1 installer file
2 longrun files
20 CLI files

### CODE REVIEW

**Comment by jyejare** [comment]
@sghai , @san7ket I had 23 files in this PR but it shows only one .... i dont know where all of them went :(

**Comment by jyejare** [comment]
@san7ket , I lost all the

File: tests/foreman/cli/test_puppetclass.py [hunk 1]
```diff
@@ -24,8 +24,9 @@
 from robottelo.cli.puppet import Puppet
 from robottelo.constants import CUSTOM_PUPPET_REPO
 from robottelo.decorators import (
+    run_only_on,
     tier2,
-    run_only_on
+    upgrade
```
Comment (sghai) [inline tests/foreman/cli/test_puppetclass.py]: 👍

**Comment by jyejare** [comment]
@san7ket , @sghai I hard reset to the commit having 23 files,  and now I got them all back.

Resolution:
```
-> git reflog
-> Select the commit to which you want to revert to
-> git reset --hard <commit-id>
-> git push origin <branch_name>
```

**Comment by sghai** [comment]
One more ACK

**Comment by san7ket** [comment]
ACK

**Pull request merged**